shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -269,9 +269,14 @@ public void removeOnFirstFrameRenderedListener(@NonNull FlutterUiDisplayListener
   @Override
   protected void onConfigurationChanged(@NonNull Configuration newConfig) {
     super.onConfigurationChanged(newConfig);
-    Log.v(TAG, "Configuration changed. Sending locales and user settings to Flutter.");
-    sendLocalesToFlutter(newConfig);
-    sendUserSettingsToFlutter();
+    // We might get invoked here before attaching again when coming back from
+    // the app switcher. In that case, we'll call these methods when we get
+    // reattached to the engine.
```

Comment:
Be more explicit here plz. Something like we've observed on Android version X that the sequence of calls can turn out to be Y, Z, W despite the JavaDoc for Z stating V. No-op'ing here because we expect this to be called again in another UI component lifecycle callback U.

Reply:
Updated.